portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,31 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+    comments.add("Awesome William!");
+    comments.add("Nice site!");
+    comments.add("reeeeeeeeeeeeeeeee");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = convertToJson(comments);
+
     response.setContentType("text/html;");
```

Comment:
Since we are returning JSON instead of HTML, it probably makes sense to set content type to "application/json".

https://stackoverflow.com/questions/404470/what-mime-type-if-json-is-being-returned-by-a-rest-api

Reply:
Got it!